src/action-ros-ci.ts

```diff
@@ -377,10 +325,9 @@ async function run_throw(): Promise<void> {
 		{ ...options, silent: true }
 	);
 
-	const curlFlags = curlFlagsArray.join(" ");
-	for (const vcsRepoFileUrl of vcsRepoFileUrlListResolved) {
+	for (const vcsRepoFileUrl of vcsRepoFileUrlListNonEmpty) {
 		await execBashCommand(
-			`curl ${curlFlags} '${vcsRepoFileUrl}' | vcs import --force --recursive src/`,
+			`vcs import --force --recursive src/ --input ${vcsRepoFileUrl}`,
```

Comment:
something is wrong here, because previously we were using a full path to the file that was retrieved using `path.resolve()` (which I guess uses the root of the git repo as a CWD so the path relative to the root of the git repo works fine), e.g.:

```
'file:///home/runner/work/action-ros-ci/action-ros-ci/.github/workflows/repo_file_for_test.repos'
```

but when we're running the command here, we're actually in `cwd = rosWorkspaceDir == /home/runner/work/action-ros-ci/action-ros-ci/ros_ws` because of the `options` object.

I'm guessing we should thus use `${workspace}/${vcsRepoFileUrl}` assuming `workspace == /home/runner/work/action-ros-ci/action-ros-ci`. But `vcsRepoFileUrl` could be a URL as well. Seems like we need to keep `resolveVcsRepoFileUrl()` after all. I guess this should work: `--input ${resolveVcsRepoFileUrl(vcsRepoFileUrl)}`.